corehq/apps/users/models.py

```diff
@@ -1311,11 +1311,6 @@ def create(cls, domain, username, password, email=None, uuid='', date='',
         else:
             couch_user = cls()
 
-        if date:
-            couch_user.created_on = force_to_datetime(date)
```

Comment:
looked into this gem a bit. couldn't immediately find any usages. it was added back in 2011 for [this](https://github.com/dimagi/commcare-hq/commit/e0073d1d5#diff-16ce9f2eb3be0695a91eca75714bb3b7R359) which is no longer in the code base